docs/api/RESTAPI.md

```diff
@@ -1,5 +1,5 @@
 <!--
-This file was generated by `sbt generateRestApiDocs` on Mon, 31 Jan 2022 15:23:32 -0500
+This file was generated by `sbt generateRestApiDocs` on Sat, 30 Apr 2022 16:27:14 -0400
```

Comment:
Not reviewing this generated file.

Incidentally, this led me down a mini-rabbit-hole. I think you can mark this file as generated in a `.gitattributes` file so that github won't expand it by default (or count the generated-lines-changed) in code reviews (see: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github)

Reply:
I think for us this would mean adding a new line to the end of `.gitattributes`:

```gitattributes
...
docs/api/RESTAPI.md linguist-generated=true
```